controller.js

```diff
@@ -1,7 +1,89 @@
-var Calculator = require( './Calculator' )
-var clickHandler = document.getElementById( 'calculator-keypad' )
+var Calc = new Calculator() //eslint-disable-line
+var clickElement = document.querySelector( '.calculator-keypad' )
 
-clickHandler.addEventListener( '.click', function( event ) {
-  var value = event.target
-  console.log( event.target )
-} )
+clickElement.addEventListener( 'click', calculatorHandler.bind( this ) )
+document.addEventListener( 'keypress', calculatorHandler.bind( this ) )
+
+function calculatorHandler ( event ) {
+  var displayElement = document.querySelector( '.calculator-display' )
+  var eventKey
+
+  if ( event.key ) {
+    eventKey = parsedKey( event.key )
+  }
+
+  var value = ( !event.key ) ? event.target.value : eventKey
+  var calculatorButton = document.querySelector( `[value='${eventKey}']` )
+
+  if ( eventKey !== null && eventKey ) {
```

Comment:
Using `setTimeout` is a cool way to do a manual animation in JS. A cleaner way to do this would be with classnames instead of injecting styles directly into your JS. Just add a className and then use setTimeout to remove that className in X time.

Try to avoid putting style information in your JS. Always try to keep that in your CSS